operations/mimir/memberlist.libsonnet

```diff
@@ -45,6 +45,11 @@
     memberlistConfig:: {
       'memberlist.bind-port': gossipRingPort,
       'memberlist.join': 'dns+gossip-ring.%s.svc.cluster.local:%d' % [$._config.namespace, gossipRingPort],
+
+      // Increase the leave timeout to being 10x the connection timeout, so that we can communicate
+      // the leave to at least 1 node, if the first 9 we try to contact times out.
+      'memberlist.leave-timeout': '20s',
+      'memberlist.packet-dial-timeout': '2s',
```

Comment:
I'm fine to do this in Jsonnet first, but I think we should really change the default config in the code.

Reply:
Let's do that. https://github.com/grafana/dskit/pull/209

Reply:
Done, ptal.